homeassistant/components/homeassistant_alerts/__init__.py

```diff
@@ -170,6 +176,23 @@ async def _async_update_data(self) -> dict[str, IntegrationAlert]:
                     if self.ha_version >= resolved_in_version:
                         continue
 
+            if self.supervisor and "supervisor" in alert:
```

Comment:
You're checking `self.supervisor` now for each alert. Maybe better to store it once outside the loop.

Reply:
sure

Reply:
https://github.com/home-assistant/core/commit/257a3d04fd8f8a4af862dc3e1477482adb32d905